gpiozero/spi_devices.py

```diff
@@ -196,43 +279,44 @@ def __init__(self, channel=0, differential=False, **spi_args):
         super(MCP33xx, self).__init__(channel, 12, differential, **spi_args)
 
     def _read(self):
-        # MCP3304/02 protocol looks like the following:
+        if self.differential:
+            result = self._words_to_int(
+                self._spi.transfer(self._send())[-2:], self.bits + 1)
+            # Account for the sign bit
+            if result > 4095:
+                return -(8192 - result)
```

Comment:
Erm ... possibly? It'd probably make it more obvious that it's **12 and **13 respectively (which ties into the bit-ness of the ADC in question). Yeah, good idea.